### ISSUE TITLE
Change the text of the warning popup of the individual reset button

### ISSUE THREAD

**Issue by Gylotip** [freeCodeCamp/freeCodeCamp#55713]
### Is your feature request related to a problem? Please describe.

The problem is that I thought this button would reset the whole course (all steps instead of just one):

![individual_reset_button_warning_popup](https://github.com/user-attachments/assets/2b41aaca-635d-4324-ba13-e3db73d2e574)

This was the reason why I was hesitant to use the reset button. I did not know that the button would reset a specific step and not all steps.

### Describe the solution you'd like

The solution is to change the text of the warning popup from this:

>Are you sure you wish to reset this lesson? The editors and tests will be reset.

To this:

>Are you sure you wish to reset step 8? The editors and tests will be reset.

The text "this lesson" is changed to "step 8" (use other numbers for other steps automatically). In my opinion, this is the best approach.

### Describe alternatives you've considered

The alternative was to make it clear in the warning popup that other steps will not be affected. However, it is unnecessary to mention that other steps will not be affected. Instead, only the specific step should be mentioned that will be reset.

**Comment by nikhilbansall** [comment]
\attempt#55713

**Comment by nikhilbansall** [comment]
\attempt #55713

**Comment by ParthShah12001** [comment]
can you assign this to me I am new here and this looks like a easy starting point and also can someone help with set up i tried a lot but i am getting few error while running it.

**Comment by Sembauke** [comment]
I don't think this is necessary. The code isn't saved in different steps anyway.

**Comment by gylotip** [comment]
@huyenltnguyen

Hmm, my suggestion was to replace `this lesson` with `step 1` (or other numbers for other steps), not with challenge names. For certifications, the reset dialog should default to `step 1` because certifications only have one step. So, I did not mean to use challenge names. I only meant the actual step numbers.

**Comment by gylotip** [comment]
@huyenltnguyen

Okay, then only challenges that use step numbers should have different reset dialogs. For challenges that use full names, the current `this lesson` reset dialog should be kept.

New users will start with Responsive Web Design (not legacy version), which only uses step numbers. This will help users understand that the reset button only affects the current step. When users encounter challenges with full names later, they will already be familiar with the concept that the reset button only affects the specific challenge.

**Comment by gylotip** [comment]
@huyenltnguyen

It is okay. I understand it, so keep things as it is.
